flow-template/locales/fr.json

```diff
@@ -0,0 +1,8 @@
+{
+  "template": {
+    "title": "Mon modèle impressionnant!",
+    "description": "Un modèle qui aide à augmenter la productivité des marchands",
+    "preInstallNote": "Vous devez désactiver la capture de paiement automatique dans l'administration de Shopify avant d'utiliser ce modèle",
+    "postInstallNote": "Veuillez vous assurer que vous avez désactivé la capture de paiement automatique dans l'administration de Shopify avant d'utiliser ce modèle"
+  }
```

Comment:
these are not used anywhere right?

Reply:
These are passed through to flow and used there.